cpp/include/cudf/partitioning.hpp

```diff
@@ -88,6 +88,8 @@ std::pair<std::unique_ptr<table>, std::vector<size_type>> hash_partition(
   table_view const& input,
   std::vector<size_type> const& columns_to_hash,
   int num_partitions,
+  hash_id hash_function               = hash_id::HASH_MURMUR3,
+  rmm::cuda_stream_view stream        = rmm::cuda_stream_default,
```

Comment:
You need to `#include <rmm/cuda_stream_view.hpp>`

Reply:
Added in https://github.com/rapidsai/cudf/pull/6726/commits/a2571ba93df4c104d62b833d2ee13205c70f31e9.